tests/functional/Teradata/ToFinal/SqlBuilderTest.php

```diff
@@ -615,7 +614,7 @@ public function testGetUpdateWithPkCommandRequireSameTables(): void
 
         $expectedSql = sprintf(
         // phpcs:ignore
-            'UPDATE %s FROM (SELECT a."col1",a."col2" FROM (SELECT "col1", "col2", ROW_NUMBER() OVER (PARTITION BY "col1" ORDER BY "col1") AS "_row_number_" FROM %s.%s) AS a WHERE a."_row_number_" = 1) "src" SET "col1" = "src"."col1", "col2" = "src"."col2" WHERE %s."col1" = "src"."col1" AND (%s."col1" <> "src"."col1" OR %s."col2" <> "src"."col2")',
+            'UPDATE %s FROM (SELECT a."col1",a."col2" FROM (SELECT "col1", "col2", ROW_NUMBER() OVER (PARTITION BY "col1" ORDER BY "col1") AS "_row_number_" FROM %s.%s) AS a WHERE a."_row_number_" = 1) "src" SET "col1" = "src"."col1", "col2" = "src"."col2" WHERE TRIM(%s."col1") = TRIM("src"."col1") AND (%s."col1" <> "src"."col1" OR %s."col2" <> "src"."col2")',
```

Comment:
Hele, proc ten `TRIM` neni i v tech dalsich podminkach? Netvrdim, ze je to spatne, jen se ptam :-)
> `AND (%s."col1" <> "src"."col1" OR %s."col2" <> "src"."col2")`

Reply:
kdyz se podivas na posledni commit ,tak tam se ty podminky odstranuji. Rano jsem to resil se zajcou, ze bud tam ten trim dame nebo to odstranime a dosli jsme k tomu, ze jsme to zrusili. Na BQ a TD se ta druha cast podminky davat nebude

Reply:
jo, uz to vidim 👍